### PR TITLE
Enhancement: Normalize composer.json

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] normalizes `composer.json`

💁‍♂️ For reference, see https://github.com/localheinz/composer-normalize.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,34 +1,38 @@
...
-        "psr-4": {
-            "Maxbanton\\Cwh\\": "src"
-        }
-    },
-    "minimum-stability": "stable",
     "require": {
         "php": ">=5.6.0",
-        "monolog/monolog": "^1.20",
-        "aws/aws-sdk-php": "^3.18"
+        "aws/aws-sdk-php": "^3.18",
```
[the remark this answers is not in the document]
Comment (localheinz) [reply composer.json]: Ha, it does a bunch of things, mostly restructuring, sorting packages - take a look at https://localheinz.com/blog/2018/01/15/normalizing-composer.json/!

[the remark this answers is not in the document]
Comment (maxbanton) [reply composer.json]: 😆 
Btw, nice lib! Thanks!

**Comment by localheinz** [comment]
Thank you, @maxbanton!

File: composer.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (localheinz) [reply composer.json]: Glad you like it, @maxbanton!

**Pull request merged**